pwndbg/commands/context.py

```diff
@@ -70,17 +75,81 @@ def validate_context_sections():
 
 class StdOutput(object):
     """A context manager wrapper to give stdout"""
-    def __enter__(*args,**kwargs):
+    def __enter__(self):
         return sys.stdout
-    def __exit__(*args, **kwargs):
+    def __exit__(self, *args, **kwargs):
+        sys.stdout.write('\n')
```

Comment:
This line will add extra empty lines.
![emptylines](https://user-images.githubusercontent.com/16643669/75645461-eec9e700-5c80-11ea-960e-55a892c857bf.png)

Reply:
Yes, was able to reproduce it (when not using clearing screen).
Working on it.